app/views/curation_concerns/base/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<!--https://cits.artic.edu/redmine/issues/2183-->
+<!--had to override this sufia template b/c we needed to override the h1/header using the asset_h1-->
```

Comment:
careful, looks like you changed the name of the method from asset_h1 to custom_h1